chacra/tests/controllers/repos/test_repos.py

```diff
@@ -13,7 +13,7 @@ def test_repo_exists(self, session):
         )
         repo.path = "some_path"
         session.commit()
-        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/api/')
+        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/')
```

Comment:
That's fine, I just wanted to confirm the intent.  The three controller methods for the ``api/`` endpoint still exist in ``controllers/repos/__init__.py`` though. We should probably remove those to avoid confusion.